src/Ansatz/Chain.jl

```diff
@@ -447,7 +447,7 @@ end
 
 Applies a local operator `gate` to the [`Chain`](@ref) tensor network.
 """
-function evolve!(qtn::Chain, gate::Dense; threshold = nothing, maxdim = nothing)
+function evolve!(qtn::Chain, gate::Dense; threshold = nothing, maxdim = nothing, iscanonical = false)
```

Comment:
mmm I'm not sure about the `iscanonical` argument name. Let's discuss it offline.